docs/content/dev/getting-started.md

```diff
@@ -1,60 +1,138 @@
 # Getting started
 
-This document will help you with setting up the development environment for HedgeDoc 2.
-
-## Preparation
-
-### Preparing the directory
+To run Hedgedoc 2.0 you need three components: the backend, the frontend and the reverse proxy.
+
+Backend and Frontend are included in the [HegdeDoc repo](https://github.com/hedgedoc/hedgedoc). The reverse proxy can be chosen by preference. For development, we
+recommend caddy and the provided configuration.
+
+## Quick guide for development setup
+
+This describes the easiest way to start a local development environment. For other deployments follow the description
+below.
+To run Hedgedoc 2.0 you need three components: the backend, the frontend and the reverse proxy.
+
+Backend and Frontend are included in the [HegdeDoc repo](https://github.com/hedgedoc/hedgedoc). The reverse proxy can be chosen by preference. For development, we
+recommend caddy and the provided configuration.
+
+## Quick guide for development setup
+
+This describes the easiest way to start a local development environment. For other deployments follow the description
+below.
+
+1. Install Node.js (at least Node 14, we recommend Node 18) and [Yarn](https://yarnpkg.com/getting-started/install)
+2. Install Caddy (select one of the two options)
+   - [Download](https://caddyserver.com/) and place the `caddy` binary in `hedgedoc/dev-reverse-proxy`. Ensure it is executable with `chmod +x caddy`
```

Comment:
If you're on MacOS you also need to flag the file as "secure" because it has been downloaded from the internet and isn't signed.. There is a UI click way to do this, but the easiest way is to execute `xattr -d com.apple.quarantine ./caddy` in a terminal. If we have a extra hint for the executable flag then we should also mention this :shrug: